app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -182,6 +184,8 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
       $scope.emsCommonModel.openstack_infra_providers_exist = data.openstack_infra_providers_exist;
       $scope.emsCommonModel.default_api_port                = '';
       $scope.emsCommonModel.amqp_api_port                   = '5672';
+      $scope.emsCommonModel.event_stream_selection          = data.event_stream_selection;
```

Comment:
Why was this line added?
After your last update, `event_stream_selection` is now only applicable to OpenStack isn't it?
So ideally, there should be no `event_stream_selection` specific code in this PR.